misc/master.sql

```diff
@@ -319,7 +321,10 @@ CREATE TABLE `storageUploadQueue` (
   `charset` varchar(25) DEFAULT NULL,
   `time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   PRIMARY KEY (`uploadKey`),
-  KEY `userID` (`userID`)
+  KEY `userID` (`userID`),
+  KEY `libraryID` (`libraryID`),
+  KEY `key` (`key`),
```

Comment:
No index on `key`.

Would also be good to include the schema-change commands (these indexes and the FK below) in a commit message. (If you put those in a new commit, I can add them to the squashed final one.)